packages/core/src/components/TextField/Styled/InnerWrapper/fusion.styled.tsx

```diff
@@ -23,15 +24,25 @@ export const fusionStyle = ({ fusion, disabled, isTextPresent }: InnerWrapperPro
             pointer-events: none;
             background-color: transparent;
             transition: all 100ms ease-out;
-            border-radius: 1rem;
+            border-radius: ${fusion.default.borderRadius};
+            border-width: ${fusion.default.borderWidth};
             border: 0.1rem solid ${fusion.default.borderColor};
         }
 
-        &:hover::after,
-        &:focus-within::after {
-            border-width: ${!disabled && `0.15rem`};
+        &:hover::after {
+            border-width: ${!disabled && fusion.hover.borderWidth};
             border-color: ${!disabled && fusion.hover.borderColor};
         }
+        
+        &:focus-within::after {
+            border-width: ${!disabled && fusion.active.borderWidth};
+            border-color: ${!disabled && fusion.active.borderColor};
+        }
```

Comment:
Same comments as below about `!disabled` and possible missing `border-radius`

Reply:
Updated, similar to the outlined styled filed.